WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -555,6 +574,10 @@ private extension OrderDetailsViewController {
                     // To be implemented.
                 })
             case .success(let receiptParameters):
+                self.syncOrderAfterPaymentCollection {
```

Comment:
After the order has been refreshed, we call the method that checks that whether the order is eligible for card present payments, which in our case would hide the "Collect Payment" button

Reply:
Nice